lib/kount/Response.rb

```diff
@@ -327,14 +327,15 @@ def get_error_count
       @paramlist['ERROR_COUNT'].to_s
     end
 
-    def geterrors
+    def get_errors
       errors = []
-      error_count = get_error_count
-      (0..error_count.to_i - 1).each do |i|
-        errors = @paramlist["ERROR_#{i}"]
+      error_count = get_error_count.to_i
+      (0..error_count - 1).each do |i|
+        errors << @paramlist["ERROR_#{i}"]
       end
       errors.compact
     end
+    alias_method :geterrors, :get_errors
```

Comment:
in case, some existing code uses `geterrors`, but since it's currently erroring maybe this is unwarranted.